.changeset/plenty-spiders-act.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': patch
+---
+
+Fixes an issue where a slot rendered multiple times resulted in the same output each time.
```

Comment:
```suggestion
Fixes an issue where rendering multiple slots resulted in the same output each time.
```
super-nit: "a slot" rendered multiple times makes me think it's the same one slot, so the output should be the same! Would this be a correct description? (non-blocking, though)

Reply:
It is the same one slot.

Reply:
Now that you bring it up though, I think there is room for clarification. Before this PR, the first render was reused when users tried to manually render the same slot multiple times.

Reply:
OK, then maybe something like.... (wordy, but trying to see if this is the right idea)

> Fixes an issue where a slot rendered multiple times incorrectly used the same initial rendering for each instance